dav1d-sys/build.rs

```diff
@@ -23,7 +23,7 @@ mod build {
 
     pub fn build_from_src(
         lib: &str,
-        version: &str,
+        _version: &str,
```

Comment:
probably would be a good idea to extend further system-deps to accomodate source tags?

Reply:
Here you could the version requirements passed in so you can theoretically build a different version or build it differently or so